contracts/axelar-gateway/src/test_gateway.rs

```diff
@@ -414,22 +415,17 @@ fn rotate_signers_bypass_rotation_delay_fail_if_not_operator() {
     let proof = generate_proof(&env, data_hash.clone(), signers);
     let bypass_rotation_delay = true;
 
-    client
-        .mock_auths(&[MockAuth {
-            address: &user,
-            invoke: &MockAuthInvoke {
-                contract: &contract_id,
-                fn_name: "rotate_signers",
-                args: (
-                    new_signers.signers.clone(),
-                    proof.clone(),
-                    bypass_rotation_delay,
-                )
-                    .into_val(&env),
-                sub_invokes: &[],
-            },
-        }])
-        .rotate_signers(&new_signers.signers, &proof, &bypass_rotation_delay);
+    assert_invoke_auth!(
```

Comment:
make the usage more natural similar to what I suggested, the macro can parse the input as a function call and figure out how to add the mock auth based on it. You can even ask Claude to generate the macro for you given the usage and the previous code that needs to be generated

Reply:
I agree that the usage should be more natural. updated the macro and code, please take look.